advanced/stack-overflow-detection/Cargo.toml

```diff
@@ -5,7 +5,7 @@ edition = "2021"
 license = "MIT OR Apache-2.0"
 
 [dependencies]
-hal             = { package = "esp32c3-hal", version = "0.13.0" }
-esp-backtrace   = { version = "0.9.0", features = ["esp32c3", "panic-handler", "exception-handler", "print-uart"] }
-esp-println     = { version = "0.7.1", features = ["esp32c3"] }
+esp-hal = {  features = ["esp32c3"], git = "https://github.com/esp-rs/esp-hal.git", package = "esp-hal" }
+esp-backtrace   = { version = "0.11.0", features = ["esp32c3", "panic-handler", "exception-handler", "println"] }
+esp-println     = { version = "0.9.0", features = ["esp32c3", "uart"] }
```

Comment:
Isn't `uart` enabled by default? Or you see some value having it explicitly?

Reply:
Yes it's the default - my thought was that it's more obvious where to change it but maybe it's not necessary. I don't have a strong opinion here (and probably don't do it consistently)